statistics/rrandom.py

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
call this module random

remove the class and make the methods module functions mimicking the interface of the python random